docs/get_started/getting_started_riscv_cmake.md

```diff
@@ -38,7 +38,7 @@ $ cmake -G Ninja -B ../iree-build-host/ -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COM
 $ cmake --build ../iree-build-host/ --target install
 ```
 
-Debugging note: if `IREE_LLVMAOT_LINKER_PATH` is set for targeting Android then the build above will fail, and you should run `unset IREE_LLVMAOT_LINKER_PATH`.
+Debugging note: if `IREE_LLVMAOT_LINKER_PATH` is set for targeting RISC-V then the build above will fail, and you should run `unset IREE_LLVMAOT_LINKER_PATH`.
```

Comment:
Why change from Android to RISC-V. Presumably this is still the case for Android?

Reply:
You need to set risk-v linker path . If `unset IREE_LLVMAOT_LINKER_PATH` it will use the default system linker (e.g `/usr/bin/ld`)

Reply:
In my experience, IREE_LLVMAOT_LINKER_PATH should only be set when iree-translate generates the LLVM backend target. Setting the environment variable would cause both host toolchain and target compilation to fail.

Reply:
Actually it occurs in every cross-compiled platform.

Reply:
Indeed you need to scope this variable during switching between translation to different targets. 
I am not familiar  with risk-v tooling, but if there is a canonical toolchain you can consider adding risk-v linker tool, see example here [here](https://github.com/google/iree/blob/main/iree/compiler/Dialect/HAL/Target/LLVM/internal/AndroidLinkerTool.cpp)

Reply:
Cool. I'll take look at that. Maybe we can have a patch in the future.

Reply:
A rv-specific linker tool can be useful. Right now it goes through the `UnixLinkerTool.cpp` path that also creates clang warning message w.r.t unused compile arguments.

Reply:
See also https://github.com/google/iree/blob/d496bc9d53e918c0362ddd498ded750eba6050f3/iree/compiler/Dialect/HAL/Target/LLVM/internal/LinkerTools.cpp#L37-L45
for where linker tools are selected based on the target triple